page.jsx

```diff
@@ -92,24 +92,24 @@ export default function createPage(swingsetOptions = {}) {
       : componentNames
 
     return (
-      <div className={s.root}>
+      <div className={s.sidebar}>
```

Comment:
I _think_ this is wrapping the entire page with the `s.sidebar` class. We might want to retain the `s.root` wrapper here and wrap only the sidebar content with `s.sidebar`.

Reply:
@BRKalow you are definitely right. Ran into some issues running the project example locally to see the update but the latest updates seemed to fix it. So I brought back the root class 👍🏼